_src/static/sass/utilities/_variables.scss

```diff
@@ -12,6 +13,12 @@
 
 
 // ----------------------------
+// #UNITS
```

Comment:
NOTE: moving this here since it affects more than just type size (we use the ``rem(*)`` Bourbon mixin for element/layout sizing too).